packages/contracts/test/contracts/OVM/chain/OVM_CanonicalTransactionChain.spec.ts

```diff
@@ -1150,132 +1393,6 @@ describe('OVM_CanonicalTransactionChain', () => {
         })
       })
 
-      describe('when trying to add elements which are older than already existing CTC elements', () => {
```

Comment:
I didn't actually delete this test case, I just moved it up to [line 1028](https://github.com/ethereum-optimism/optimism/pull/1156/files#diff-6d10af15ef118b447a7e7c81f447b0b7dd6d591e9cc17f3a1504273dfe912f85R1028) so that it's nested within this `describe` function:

https://github.com/ethereum-optimism/optimism/blob/b9ed07fbc52305fc0377d2e61a7cf986f00d1ee7/packages/contracts/test/contracts/OVM/chain/OVM_CanonicalTransactionChain.spec.ts#L899

You can check by searching for the description messages.